docs/createconsolescreenbuffer.md

```diff
@@ -5,6 +5,9 @@ author: bitcrazed
 ms.author: richturn
 ms.topic: article
 keywords: console, character mode applications, command line applications, terminal applications, console api
+f1_keywords:
+- WINNT/FILE_SHARE_READ
+- FILE_SHARE_READ
```

Comment:
This isn't right at all. You might have to use this flag in this API call, but this page doesn't define anything about `FILE_SHARE_READ`. It defines `CreateConsoleScreenBuffer` off the console headers.